frontend/src/index.ts

```diff
@@ -947,3 +920,10 @@ export class App extends LiteElement {
     AppStateService.updateOrgSlug(null);
   }
 }
+
+declare global {
+  interface GlobalEventHandlersEventMap
```

Comment:
@emma-sg This feels a lot simpler than how the org event map and `OrgEventListener` is typed--you still get event type completion, and devs don't need to know advanced typescript to understand what's going on. Any reason not to do the same approach in `org/index.ts`?

Reply:
Sure, I think this makes sense! Considering how events move through the DOM & could be sent from anywhere